pkg/apis/pipeline/v1beta1/taskrun_validation_test.go

```diff
@@ -378,6 +378,29 @@ func TestTaskRunSpec_Invalidate(t *testing.T) {
 			"computeResources",
 		),
 		wc: config.EnableAlphaAPIFields,
+	}, {
+		name: "invalid both step-level (step.resources) and task-level (spec.computeResources) resource requirements",
```

Comment:
could you also add a test case where the taskrun has stepoverrides?

Reply:
ya. there was a test case (above this one) I added in previous PR.